configuration/config-txt/memory.md

```diff
@@ -30,6 +30,14 @@ The `gpu_mem_512` command sets the GPU memory in megabytes for Raspberry Pis wit
 
 The `gpu_mem_1024` command sets the GPU memory in megabytes for Raspberry Pis with 1GB or more of memory. (It is ignored if memory size is smaller than 1GB). This overrides `gpu_mem`.
 
+## total_mem
+
+This parameter can be used to force a Raspberry Pi to limit its memory capacity: specify the total amount of RAM you wish the Pi to use. For example, to make a 4GB Raspberry Pi 4 behave as though it were a 1GB model, use the following:
```

Comment:
It's obvious from the example, but maybe the text should clarify that this value is specified in megabytes?

Reply:
Good catch - fixed.